src/cmr/sizing/formats.clj

```diff
@@ -63,16 +80,42 @@
     (+ (* granule-count compression measurements)
        (* granule-count metadata))))
 
-(defn- estimate-netcdf4-or-ascii-size
-  "Calculates the estimated size for ASCII or NETCDF4 format.
+(defn- estimate-ascii-size
+  "Calculates the estimated size for ASCII.
+  In ASCII output, the actual size for a granule in bytes is the sum of digit number
+  of all the dimension data + 2 bytes(comma and space) for each data except for the last one.
+  To estimate, we assume 50% dimension data has the average FillValues digit number.
+  the other 50% dimention data has the average ValidRanges digit number.
+  The estimated size = 0.5 * avg-fill-value-digit-number * dimensionality +
+                       0.5 * avg-valid-range-digit-number * dimensionality +
+                       2 * (dimensionality -1)"
+  [granule-count variables params]
+  (* granule-count  
```

Comment:
Based on the unit tests it looks like we should cast this to a long (estimates should be in bytes, no decimal numbers).